config/routes.rb

```diff
@@ -47,7 +47,7 @@
     get "/check_email_availability" => "marketplaces#check_email_availability"
   end
 
-  REMOVED_LOCALES = Rails.application.config.REMOVED_LOCALES.to_a
+  REMOVED_LOCALES ||= Rails.application.config.REMOVED_LOCALES.to_a
```

Comment:
I'd say the `REMOVED_LOCALES` is a bit unnecessary here. It's used only once in this file (line 52), so we could get rid of this variable assignment.

The `Rails.application.config.REMOVED_LOCALES` is assigned to `Sharetribe::REMOVED_LOCALES` in `application.rb`. The `Sharetribe::REMOVED_LOCALES` is defined in `available_locales.rb`.

So in the line 52, we could use the `Sharetribe::REMOVED_LOCALES`, like this:

```ruby
locale_matcher = Regexp.new(Sharetribe::AVAILABLE_LOCALES.map { |l| l[:ident] }.concat(Sharetribe::REMOVED_LOCALES).join("|"))
```

Does it make sense? Would you mind removing the `REMOVED_LOCALES` and using the `Sharetribe::AVAILABLE_LOCALES` instead?